.devcontainer/devcontainer.json

```diff
@@ -10,7 +10,6 @@
 	"postCreateCommand": "./.devcontainer/post-create.sh",
 	"customizations": {
 		"vscode": {
-			"settings": { "ms-python.flake8.args": ["--ignore=E501"] },
```

Comment:
Why have you removed this?

Reply:
This is not the way to define rules using the dev container (it doesn't work). Instead, I added a flake8 file with this rule, which will also apply to those who don't use the dev container. Since I opened this PR, another PR also added a flake8 file, so you don't see the file here.